services/core-web/src/tests/components/modalContent/ReplaceDocumentModal.spec.tsx

```diff
@@ -1,33 +1,33 @@
 import React from "react";
-import { render } from "@testing-library/react";
 import { Provider } from "react-redux";
-import { store } from "@/App";
+import configureStore from "redux-mock-store";
+import ReplaceDocumentModal from "@mds/common/components/documents/ReplaceDocumentModal";
 import { MINEDOCUMENTS } from "@/tests/mocks/dataMocks";
 import matchMedia from "@/tests/mocks/matchMedia";
-import ReplaceDocumentModal from "@mds/common/components/documents/ReplaceDocumentModal";
+import { shallow } from "enzyme";
+
+const mockStore = configureStore([]);
+const store = mockStore({});
 
 const props = {
   document: MINEDOCUMENTS[0],
   handleSubmit: jest.fn().mockReturnValue(Promise.resolve()),
   closeModal: jest.fn(),
   postNewDocumentVersion: jest.fn().mockReturnValue(Promise.resolve()),
+  alertMessage: "This is a test alert message.",
 };
 
-beforeEach(() => {
-  props.document = MINEDOCUMENTS;
-});
-
 beforeAll(() => {
   window.matchMedia = matchMedia;
 });
 
 describe("ReplaceDocumentModal", () => {
-  it("renders properly", () => {
-    const { container } = render(
+  it("renders correctly and matches the snapshot", () => {
```

Comment:
I am wondering why go back to the "shallow" render when the react-testing-library render gave us a better snapshot.